tests/client/test_base_client.py

```diff
@@ -469,7 +506,39 @@ async def test_prefect_httpx_client_raises_prefect_http_status_error(
         base_client_send.return_value = RESPONSE_400
         with pytest.raises(PrefectHTTPStatusError) as exc:
             await client.post(url="fake.url/fake/route", data={"evenmorefake": "data"})
-            assert (
-                "Response: {'extra_info': [{'message': 'a test error message'}]}"
-                in str(exc)
-            )
+        expected = "Response: {'extra_info': [{'message': 'a test error message'}]}"
```

Comment:
This is a bug fix for an existing test (the assert needs to be outside`with pytest.raises`, otherwise it's not actually executed). To verify bug, change the expected value to "XXX", move it back inside `with pytest.raises`, and note that the test does not fail.